src/System.Numerics/ref/Compat/System.Numerics.cs

```diff
@@ -27,7 +27,7 @@ public partial struct BigInteger : System.IComparable, System.IComparable<System
         public static int Compare(System.Numerics.BigInteger left, System.Numerics.BigInteger right) { return default(int); }
         public int CompareTo(long other) { return default(int); }
         public int CompareTo(System.Numerics.BigInteger other) { return default(int); }
-        public int CompareTo(object obj) { return default(int); }
+        int System.IComparable.CompareTo(object obj) { return default(int); }
```

Comment:
I think that is going the wrong direction. We need to keep these compatible with the older surface area. In other words when in doubt the implementation needs to implicitly implement these so that it will be binary compatible with people compiled against the implicit or explicit version of this API.